apps/comments/src/utils/decodeHtmlEntities.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @copyright Copyright (c) 2021 Christopher Ng <chrng8@gmail.com>
+ *
+ * @author Christopher Ng <chrng8@gmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * @param {any} value -
+ * @param {any} passes -
+ */
+export default function decodeHtmlEntities(value, passes = 1) {
```

Comment:
```suggestion
export function decodeHtmlEntities(value, passes = 1) {
```
Let's go with a named export for better IDE integration